glob/CMakeLists.txt

```
@@ -125,6 +125,12 @@ new_ec_test(braces_word_choice2 braces.in test.js "^choice=true[ \t\n\r]*$")
 new_ec_test(braces_word_choice3 braces.in test.html "^choice=true[ \t\n\r]*$")
 new_ec_test(braces_word_choice4 braces.in test.pyc "^[ \t\n\r]*$")
 
+# word choice with spaces
+new_ec_test(braces_word_choice5 braces.in test.py "^choice=true[ \t\n\r]*$")
```

If you take my suggestions in `glob/braces.in`, this should change to:
```suggestion
new_ec_test(braces_word_choice5 braces.in test.pl "^choice=spaces[ \t\n\r]*$")
```
Lines 130 and 131 should have corresponding changes.

---

I changed test case extension and correct matcher